functionary/ui/tables/build.py

```diff
@@ -1,10 +1,33 @@
+import django_filters
 import django_tables2 as tables
 from django.urls import reverse
 
 from builder.models import Build
 from ui.tables import DATETIME_FORMAT
+from ui.tables.filters import DateTimeFilter
 from ui.tables.meta import BaseMeta
 
+FIELDS = ("package__name", "status", "created_at", "creator")
```

Comment:
As is the label for package name ends up as "Package Name" rather than just "Package."  Which then lead me to notice that the table column is called "Name" rather than "Package".  We should fix both of those things.